srcpkgs/libnest2d/patches/soversion.patch

```diff
@@ -0,0 +1,21 @@
+From 12b64796aee24d5af41df3a10c4313dc713c3e74 Mon Sep 17 00:00:00 2001
+From: Karl Nilsson <karl.robert.nilsson@gmail.com>
+Date: Wed, 13 Jan 2021 21:51:34 -0500
+Subject: [PATCH] CMake: add SOVERSION to shared library
+
+---
+ CMakeLists.txt | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 374e90f..c2f2bec 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -95,6 +95,7 @@ if(NOT LIBNEST2D_HEADER_ONLY)
+     add_library(${LIBNAME} ${PROJECT_SOURCE_DIR}/src/libnest2d.cpp)
+     set_target_properties(${LIBNAME} PROPERTIES PREFIX "")
+     set_target_properties(${LIBNAME} PROPERTIES DEBUG_POSTFIX "d")
++    set_target_properties(${LIBNAME} PROPERTIES SOVERSION "1.0")
```

Comment:
Maybe best not to set this if upstream doesn't guarantee ABI?

Reply:
That patch has already been upstreamed. I'm manually patching because there's no recent release.
https://github.com/tamasmeszaros/libnest2d/pull/27